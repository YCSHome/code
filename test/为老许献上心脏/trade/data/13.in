400 500 500 
162 496574183711906
238 414328565270359
114 990392700092027
134 412483193428995
74 72902033127158
159 393227845213492
34 330029659842553
8 40899385907133
220 594460531263727
234 234762452331776
101 368784655136953
55 966543395991897
201 95768603084950
108 785209431602182
393 587853530011406
83 49396411656712
325 820058009713239
156 88996932415022
220 870672797938023
20 316281911363488
15 710636509252024
197 182541266244907
396 759775603549499
336 319782393814981
28 108502648221076
304 617280510305640
261 643798974239128
191 426372260265356
136 877705065193844
388 484395030616285
177 20202899247532
149 262905934360969
30 923457170914710
337 247495648392055
310 644337939907721
151 132148915943830
202 647674717098564
387 405408470909228
21 87000436213074
149 683982977870443
272 149650780831991
381 210224603894384
28 844120005654142
127 496587366153194
302 778220222062326
62 633413161200978
92 411663833308881
227 535886793703661
214 629027338716799
269 810234435107378
74 330977182070236
99 311907929981516
391 293157170529701
237 542545432913359
34 784715800005644
188 424138391736643
144 553110714846173
73 202489146837717
261 805179091190448
67 294241268710902
55 581748381167758
59 154643522841221
301 595878809674382
289 686005885403429
343 127245577527692
73 752922711946432
176 329102801171267
175 274187511317888
314 431141330255006
70 882284022144134
181 947292199276332
37 719550380384131
44 498421823520436
205 853830075993250
221 271880381664018
122 800061142291781
46 570574721615875
69 971119650861815
139 871884448243606
393 541351089058442
184 194599655191193
217 822741279338895
252 400778926299987
186 418513868755050
31 958902495651746
194 942518125518029
348 705503892735267
105 52323347072269
224 774278161707594
119 145981538464618
172 54017933289136
170 270335503495075
285 382259391746482
266 375044437155287
155 435622724669464
336 373734245949762
5 909731524461978
31 582343902322220
150 461999705945250
340 938694236318841
45 183338122296688
218 312566166642320
7 616795713101579
46 354014450146677
298 963780813588080
347 143826236283731
373 205242873958753
205 467377003052771
174 199712467044314
297 795741231367983
79 286848516745081
362 684836266833705
74 174008536755874
365 464791802576172
376 227250459680809
373 475584744364538
199 142101442772282
65 134153374047013
7 333142218794958
336 839053159442743
162 815559459126944
121 73912987753454
336 386684009077264
76 662284703644711
328 670153525740668
150 238896890576714
212 912635747725534
354 721891680763253
166 659672247004845
234 873392641464295
229 91576686744861
334 330256207657552
36 250871206713338
7 611440582269818
243 313388156181671
379 976140111284587
328 446424106119735
121 102666889683599
388 595484984058800
193 75390262153448
25 283269698251534
33 325161443807112
311 576242685223443
333 366816134855263
75 266628142084882
135 675871123569039
216 956255827576744
35 773696091297355
329 715715592442396
249 770221747775667
124 161152730871088
341 196253421524517
153 103688541290399
273 2309925649502
279 323059608747853
127 712171547193608
166 753206378838450
161 189045431518332
150 534061432781004
153 315997441899459
321 774590328210773
102 374034075217089
257 99511762381030
282 303853068216615
283 743260259044981
350 136885165197294
264 935700608846136
371 61140613090365
280 494460295665995
168 707761539004928
233 387018598399338
8 11967674054216
378 17518360203987
361 629057397080384
333 16561561759599
257 641235905669757
57 389353005603091
212 280112733550234
67 993437479316474
285 394710818106757
148 450613057079440
333 835336645288014
141 294448428305415
243 612726013820979
181 915291051415799
270 687342820132424
366 20339628067135
266 763634027507719
293 619341464721341
373 59934980001429
90 745148420757584
100 902616620305299
260 974940025453812
42 344195256609313
37 426137992029267
110 988163680716822
369 541464300773142
84 159558556596631
158 426635713522719
241 745692836071913
89 274309747492135
287 95797183282070
229 39551234790404
96 218127201979779
318 827874774868562
323 907975409088080
213 551475550324284
296 153609988658353
33 90385973530672
340 716158063598572
345 919465571137029
46 53727485152142
174 293863067040863
125 883601069014120
59 374456010652309
84 350848346691517
192 719212298941470
153 873689963289551
126 934407253074550
84 516547873291015
254 359582165806015
225 769410901120520
40 810611464124368
36 846220796784651
244 795195183825984
158 568331885856154
94 434502305534771
241 917421978761153
378 339814528343396
396 667134748956906
242 552879201355442
203 235996791339227
300 341699387615081
1 970214976386799
159 310613546076948
100 595490745823981
45 585291343960930
332 975617439580794
350 558799665855857
109 466680938517662
21 106562937649332
199 962574280957843
146 194106781289976
68 545022670197240
199 678174398977463
284 833493788276939
199 735238058188176
385 111942465268495
169 326244549551392
263 418918701025453
185 898111206746778
191 543061184298932
10 426111655960804
342 332741677777879
17 97614166825292
226 887447372731509
366 972366995027663
118 426351608710904
252 463743504403232
159 351072144179198
383 876581181142514
89 965274619034836
222 902002406522353
29 894973757438238
353 507403939798418
251 206138371889778
39 498649018781464
376 353321485794524
263 890838940985025
179 486602369202377
120 867215865560806
158 553364591258564
313 929470292759750
73 604567090360828
299 222840643514560
294 150802925551158
355 843214576399507
373 384917296722081
199 33693980790290
278 121014800210045
187 513059669084208
307 868568917850492
170 399831517534520
255 292731228328950
149 241785792087345
234 149431232786747
303 560996062001780
21 877224315753846
218 914736667638283
93 470117755948770
83 950151837106478
144 927045207158112
195 570381030913353
215 565442417750897
97 105363927428690
130 14509721506170
177 105844660549507
210 319521066241471
274 644450497518757
114 288066867912519
132 695424809674386
338 447897390435766
97 713546498003614
36 623988274796206
388 66815501910955
190 902179300595117
60 454645698100745
370 952023116378502
287 348202781855504
261 280917154303147
162 320742567904656
265 303211922564664
383 682155170408713
64 534829544230307
209 568512301456382
246 270751589001629
180 671450414718795
188 798949759868797
116 954353500655791
218 378720127270578
184 206028431772430
46 311643141375260
299 177070409304692
206 526628308726880
385 748889975452921
301 356332891023535
104 924507770260296
63 347788372720315
69 572852326061092
287 948374836537025
392 469723658160618
165 940825078628141
222 299725543563874
148 982572306823315
86 317815350438857
224 540437598583538
332 64098219983981
349 922119505440508
394 613239949236597
198 161736382646749
122 757445572656367
225 589973174523161
275 413509908255303
300 355099448839455
376 904291147366842
272 539543514508768
284 772998542627782
262 843231173540506
338 965647452704159
383 36028086122865
89 175246635880629
89 930448508369496
396 132577458653821
329 181635737184061
14 455743494272144
305 867853434626094
163 667748758828785
337 176238271349867
306 492453462151445
75 934476948496254
312 39110951225241
249 447054249413279
350 525749285886746
93 601153340198463
392 204859325996694
210 389521883268776
249 442799929077523
12 631924091940276
65 171397665159430
196 692572209387667
72 352388575934964
158 851068397838689
206 994704013265177
180 999288194248606
203 455924769568748
118 667874734717125
273 208568635968766
39 197671315445069
392 25912118608386
107 345130821381644
43 692252835188611
156 174498814937871
351 32428512988198
260 715686753059527
133 198706466691247
36 436824877889517
104 119756372750263
109 737474815436594
223 709549363488175
256 521912496582344
93 603460596674440
231 837961933796469
40 910478619308609
137 591867499425900
383 521083477565422
57 948724258442624
255 500563328025398
213 877182772685140
124 601828228305673
259 922054877589177
10 648777664947186
312 185165010190157
44 187363464319308
114 129842074767396
368 272834486051189
139 622093564781948
245 357881103047527
190 27671485121730
345 912943648776602
281 493384053480299
31 971839797335953
150 292837442545389
331 708728993054806
363 324143122542317
22 970597662257157
85 567165838398945
114 370149814845954
25 231650162123422
223 971011165711304
398 639954090879462
248 393532056610698
257 535345451135389
235 448964630819901
126 508705297618557
393 511266813476075
129 916769202220686
120 943906668817688
49 763137168678563
24 238289741234786
200 694851909011854
162 668643851900354
25 67124763173627
138 796571865807530
399 831471344017179
194 125402547936837
190 455332328659843
301 728736325563400
176 290147879579129
62 332566023245454
212 712969294459729
23 417604676384332
362 387232240791761
288 348416170133596
290 378562594344939
13 979397325227206
246 807216574667743
1 153892831888802
365 238565307299353
35 295220728384945
369 737695626409240
257 586271000673407
119 163834685859448
101 417046892852071
94 103713423514679
243 106569454195420
300 770973019814387
24 81149927406027
24 993810021870200
41 394121292550226
168 219421339710434
240 67445402741243
36 220455652776451
156 474499875269499
387 886856717354809
303 57118442637595
31 17729374406652
263 343665350524526
285 567616615553922
257 87853904703169
179 61541546591786
49 25639186058581
398 982884356198635
49 665467280122942
214 687832831870766
392 767145663328541
77 755067656639978
148 998846837705434
386 949705877518137
177 619753688779046
275 992678211908448
103 179706646229176
194 924996851510839
199 389584962116870
123 268190418447098
303 226873462159753
67 247576154573332
162 587689407391441
329 989376435198988
69 154051949607183
158 314030304483121
329 901414276484664
203 819809139545260
360 983815290855301
37 189978327499073
148 741821965077676
341 382664394617563
297 954002341364796
234 581971480732021
373 330829408669503
184 197025310402311
397 68358616711643
207 669433232150957
45 404128384953389
291 492184628093015
238 176307480944424
211 896447456980670
146 921030799856185
106 960072014609875
222 970633433625957
62 30151811847869
110 374281358579233
369 905877217756194
225 191436257073813
37 271273051788384
197 824326055432490
275 439359270267703
246 841744797983223
143 842085888931306
370 548089288059625
365 465623349961758
327 374267214273164
205 445768069416076
119 342703895274989
338 459732810297968
351 260015253942670
103 527107228314141
107 47951595310810
20 994566263847154
335 867006940293398
280 924646317037493
275 980031327052767
171 60606767592045
305 41445294215539
63 171354832771490
303 482977151961227
320 412933701325625
109 198971075427612
247 393562635473885
4 824598923858673
337 47011434006053
22 411915159712079
313 439535875521371
168 709443085535331
297 249398419334023
307 547529868006847
305 139313766824095
320 544274130402413
210 734441014605983
53 543417478900956
58 380616843433253
123 39910454142354
273 577230251596637
147 510946680317962
181 781296602106404
356 849396258967753
159 895069778344506
297 748782189823187
375 418914084155984
347 186146640225294
89 805146893952744
313 242323957191177
133 916392534054538
127 750136684316440
287 182461535030333
394 331262895540468
336 356487928537529
318 698224402814314
363 776627833276078
256 225660279019269
31 556921071367432
17 635391425033935
178 33818322067711
110 220871829295349
290 895421087115113
395 314199395820594
363 367023102585419
294 922738167509056
329 285899229411806
130 443385683522823
83 353473052870596
108 346888841799941
373 469728650943085
64 886478324326314
285 703992362302617
367 898550909990671
265 828048496118578
279 818248098010114
255 981924534015833
90 395001770061226
138 300395624076274
375 358481804483980
185 48296307120635
124 390402669194823
304 885463548720189
314 451165942845323
322 768481893173320
249 831536692095550
271 495037804547716
128 696888008490408
52 288274433539553
289 440850926489664
200 834960654500544
255 469629402168911
249 82327541663803
338 317721623033730
111 286221115987043
108 966127208032623
144 634925508284896
8 808400868815774
9 251699455234462
39 387827986466253
323 408447477553366
262 62727667880464
277 231337314556641
99 909108128634972
13 632791589403498
38 290290221054844
287 959132196042275
72 639348140774163
312 202463763760859
326 812091739474797
121 746615004130494
267 532106704152990
175 108118014403425
71 868091577972453
146 61339862351470
82 671800741541759
376 293111653418861
264 205888647934044
357 235223954365275
9 861872926202932
84 136232945645617
232 135585838117351
99 998006460721747
206 966587406140375
148 44787812177920
204 142745414018984
306 274401706539035
276 344301293739646
192 628051744048823
364 690808109961351
199 546807613597799
277 718819309051445
256 553334899188599
381 420274382768099
84 36276649402087
154 287624727959897
264 587276520773366
165 68402532022600
108 511084993239173
215 337827158402919
173 679479621076320
328 825749101030749
396 474759619525745
378 701016401330246
263 91684051927575
151 530210581539629
35 948912034533916
49 667252527203110
3 868542618158016
119 75783059775121
190 571835654173927
288 395541760182657
96 643597784666679
209 257932976857825
84 436553802121999
206 984656706600096
139 34133473862043
321 245423882502554
23 668168933168239
190 283333324535268
49 996035419372202
93 397612860770342
195 805924811219135
99 33255970639942
282 178124047548462
234 716486162440881
294 566956502625419
297 83705781691177
226 174152841988347
65 937853471270432
355 43681363510578
139 479621787266519
353 896265609256097
83 139935851540795
325 574348330504511
238 452566504472188
131 10637957847820
359 31020564473755
141 141673066536097
158 919282760316601
172 959740501788279
256 101424089749417
278 438449791604087
310 126567778579507
76 865661854920885
14 706112921225449
371 217902326407338
57 145257953137559
151 629907482898900
333 93098128228813
45 961301726372747
262 214304770553716
174 244514750207894
398 390228797337698
314 949065497323145
335 65712268432017
201 395720562218046
343 945184493782863
154 120984006843384
87 515294715258781
59 503507846113293
287 335956016212118
10 510233608862340
73 175741117968995
73 349554190831381
305 801215433931144
143 860499833187605
317 412520455875250
288 843413868332522
143 907551164944726
128 768147025647623
55 806895233696226
386 894876485460631
30 705521180470080
362 657685607692541
301 688483413049282
261 581323864577512
164 46848729914957
249 896787379189989
398 325069748442671
58 204971285588484
222 760892854398416
4 76362731890471
296 37643939493582
365 330234326409231
107 248271509666533
311 665543565736037
214 867959303749951
397 2586763254520
330 334777249444408
387 272668040528039
124 116978332071126
227 433274854317098
335 873667916148517
42 994413954209980
41 451302288994796
7 306981029351926
316 208165569343635
217 802920747355690
124 788324109005398
188 384519610031157
171 892920504109273
247 964952744954037
210 836107206617150
366 307212174018855
153 825879573961258
59 185494664391553
400 924378553140382
79 484598784594309
334 698301657061731
131 166280985784696
144 806905330126994
30 620446082131436
400 395722027145288
15 503596798903335
112 778343253387694
339 691366145374575
336 657728891074752
212 152661345179163
220 947054356314521
206 984597609404802
115 446719015641557
196 869928950939021
356 683774178032783
192 813351342892385
272 793057758113305
146 941916858207936
44 537996503312267
220 140009651101744
131 508512089349034
144 960497772161685
219 237262987468900
389 838903276525359
360 117429788306242
164 352358413792177
24 502919063613201
231 929973377431538
323 625080221691568
29 872678434303148
304 87146290332532
154 10891468117443
186 537865133957639
390 580385818389416
276 899680575761434
363 596623093657836
211 828207601251511
272 418184660550179
135 166030837615858
366 723489237239216
