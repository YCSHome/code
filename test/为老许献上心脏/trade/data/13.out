7850484950013421
